community subscription: expand supported services (#101)
NethServer/nethsecurity-docs: subscription.rst

which provides additional benefits and features:

- **Community subscription**: self-service registration mostly suited for IT consultant.
  For more info, see the `subscription plans <https://nethsecurity.org/subscription>`_ page.

  Community subscription portal is available at `my.nethserver.com <https://my.nethserver.com>`_

> - **Enterprise subscription**: service reserved to Nethesis resellers, request more info at `sales departement <mailto:info@nethesis.it>`_.

Reviewer: ```suggestion
- **Enterprise subscription**: service reserved to Nethesis resellers, please contact the `sales department <mailto:info@nethesis.it>`_.
```
